packages/runtime/fixtures/monorepo/serviceApp/platformatic.service.json

```diff
@@ -14,6 +14,7 @@
   },
   "clients": [
     {
+      "id": "with-logger",
```

Comment:
The name `id` in this context implies to me that it's a client ID. How about `serviceId` to be more explicit?

Reply:
No preference from me. @mcollina?

Reply:
let's use `serviceId`

Reply:
Done